bloom-instance/vars.tf

```diff
@@ -57,4 +57,9 @@ variable "sdlc_stage" {
     condition     = contains(["dev", "test", "qa", "staging", "prod"], var.sdlc_stage)
     error_message = "Valid values for var: sdlc_stage are (dev, test, qa, staging, prod)."
   }
-}
\ No newline at end of file
+}
+
+variable "vpc_cidr" {
```

Comment:
Question -- the #7 says to put vpc_cidr in a variable. However, there is def some duplication here since the same var gets defined twice, once in ./vars.tf and once in ./network/inputs.tf. Seems crufty? Is there a better pattern to do this? If not its not a big deal.

Reply:
It's fairly common to have some vars defined in the main vars file that get passed into modules (hence requiring the same var to be defined once for each scope), though sometimes they may get modified along the way such as by interpolating into more specific/meaningful strings 
 or through lookups.  In this particular case, the network components don't need to be in a module, so you can just add the var to the main vars.tf file.